pkg/scheduler/framework/plugins/noderesources/balanced_allocation_test.go

```diff
@@ -291,16 +297,90 @@ func TestNodeResourcesBalancedAllocation(t *testing.T) {
 			args:        config.NodeResourcesBalancedAllocationArgs{Resources: defaultResourceBalancedAllocationSet},
 			runPreScore: false,
 		},
+		{
+			// Node1 scores on 0-MaxNodeScore scale
+			// CPU Fraction: 3000 / 35000 = 0.8571
+			// Memory Fraction: 0 / 40000 = 0
+			// DRA Fraction: 1 / 8 = 0.125
+			// Fraction mean: (0.8571 + 0 + 0.125) / 3 = 0.3274
+			// Node1 std: sqrt(((0.8571 - 0.3274)**2 + (0- 0.3274)**2 + (0.125 - 0.3274)**2) / 3) = 0.378
+			// Node1 Score: (1 - 0.378)*MaxNodeScore = 62
+			// Node2 scores on 0-MaxNodeScore scale
+			// CPU Fraction: 3000 / 35000 = 0.8571
+			// Memory Fraction: 5000 / 40000 = 0.125
```

Comment:
node1 and node2 have the same memory, why the fraction are different, node 1 memory fraction is 0, node2 is 0.125?

Reply:
Good question! This is because of the different pods scheduled to the node1 and node2. cpuOnly (see below) is scheduled to the node1, cpuAndMemory - to the node2.